Updates/0237_rp_klaven_mortwake.sql

```diff
@@ -0,0 +1,199 @@
+(@PATH,1,1,-11133.434,550.86005,70.381966,100, 0, 0),
+(@PATH,1,2,-11131.567,548.74554,70.42528,100, 0, 0),
+(@PATH,1,3,-11130.786,544.9505,70.41748,100, 0, 0),
+(@PATH,1,4,-11131.364,542.09845,70.433556,100, 0, 0),
+(@PATH,1,5,-11134.433,537.0684,70.441444,100, 0, 0),
+(@PATH,1,6,-11137.308,534.6409,70.43133,100, 0, 0),
+(@PATH,1,7,-11141.096,533.334,70.433655,100, 0, 0),
+(@PATH,1,8,-11134.433,537.0684,70.441444,100, 0, 0),
+(@PATH,1,9,-11132.151,539.1321,70.44842,100, 0, 0),
+(@PATH,1,10,-11130.786,544.9505,70.41748,100, 0, 0),
+(@PATH,1,11,-11131.174,546.92267,70.44154,100, 0, 0),
+(@PATH,1,12,-11133.434,550.86005,70.381966,100, 0, 0),
+(@PATH,1,13,-11130.786,544.9505,70.41748,100, 0, 0),
+(@PATH,1,14,-11132.151,539.1321,70.44842,100, 0, 0),
+(@PATH,1,15,-11134.433,537.0684,70.441444,100, 0, 0),
+(@PATH,1,16,-11137.308,534.6409,70.43133,100, 0, 0),
+(@PATH,1,17,-11141.096,533.334,70.433655,100, 0, 0),
+(@PATH,1,18,-11134.433,537.0684,70.441444,100, 0, 0),
+(@PATH,1,19,-11132.151,539.1321,70.44842,100, 0, 0),
+(@PATH,1,20,-11130.786,544.9505,70.41748,100, 0, 0),
+(@PATH,1,21,-11131.174,546.92267,70.44154,100, 0, 0);
+
+DELETE FROM dbscripts_on_creature_movement WHERE id IN (705301, 705302, 705303, 705304);
+INSERT INTO `dbscripts_on_creature_movement` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `dataint`, `dataint2`,`x`, `y`, `z`, `comments`) VALUES 
+('705301', '1000', '0', '1', '16', '0', '0','0', '0', '0', '0', '0','0','Klaven Mortwake - Emote Kneel'),
+('705301', '10000', '1', '0', '0', '0', '0','0', '3094', '0', '0', '0','0','Klaven Mortwake - Talk'),
+('705301', '17000', '1', '0', '0', '0', '0','0', '3095', '0', '0', '0','0','Klaven Mortwake - Talk'),
+('705301', '23000', '0', '1', '5', '0', '0','0', '0', '0', '0', '0','0','Klaven Mortwake - Emote Exclamation'),
+('705301', '23000', '1', '0', '0', '0', '0','0', '3093', '0', '0', '0','0','Klaven Mortwake - Talk'),
+('705302', '4000', '0', '0', '0', '0', '0','0', '3097', '0', '0', '0','0','Klaven Mortwake - Talk'),
+('705303', '0', '0', '0', '0', '0', '0','0', '3098', '0', '0', '0','0','Klaven Mortwake - Talk'),
+('705304', '2000', '0', '20', '2', '1', '0','0', '0', '0', '0', '0','0','Klaven Mortwake - Set WaypointPath 1');
+
+-- Defias Tower Sentry
+-- Correct Spawns and add them into spawn_group for correct movement
+-- Side note: they spawn side by side but move linear SPAWN_GROUP_FORMATION_TYPE_SINGLE_FILE
+UPDATE `creature` SET `position_x`='-11129.781', `position_y`='548.0863', `position_z`='61.728893', `orientation`='1.98334' WHERE (`guid`='66995');
+UPDATE `creature` SET `position_x`='-11131.747', `position_y`='547.9329', `position_z`='61.673286', `orientation`='1.92729' WHERE (`guid`='66994');
+
+DELETE FROM `spawn_group` WHERE `Id` = '31';
+INSERT INTO `spawn_group` (`Id`, `Name`, `Type`, `MaxCount`, `WorldState`, `Flags`) VALUES 
+('31', 'Westfall - Defias Tower Sentry', '0', '2', '0', '9');
```

Comment:
i will change spawn_group to 19003, as classic content is 1 to 19999. - `31	Western Plaguelands - Truesilver Deposit | Rich Thorium Vein (1) Ore 000` already in use
ranges currently used on and off are obv, the 1-X group which 31 is already used, so merging this creates conflicts.
i reworked fishing in from 9000 to 9056 range
tobsch is currently committing on 10000-10XXX range and the 1-100 range as he does so by MAP 1-10k for map 0 and 10k to 20k for map 1 just stay away from low numbers in both sections.
and we're using 19001 to 19999 for stuff commited to tbc-db or wotlk-db but used in classic content
<https://github.com/cmangos/issues/wiki/spawn_group>

Reply:
Okay good to know.

Reply:
btw what is flags = 9?

Reply:
i finaly had some time to look at this again, and yes flag 9 was a copy paste fail. 
Changed this and also changed spawn_group Id to 19003